### PR TITLE
Update sig-spotlight-blog-process.md

### LINKED ISSUE #7278: Update the SIG Spotlight support information
### Describe the issue

We have some minimal information there, but it would be helpful and useful to update the instructions to cover the existing best-practices, currently only available in our Slack channel. Some examples:

- Better description of the end-to-end process
- Add the information on the Blogging Team guidelines
- Better descrive the suggested Google Docs / hack.md process
- Link to other resources

In general, it should be a reference that can be used as a starting point for anyone wishing to write one.

### PR DESCRIPTION
**Pull request by fsmunoz**
Updating Spotlight blogging guidelines to be more comprehensive and include information discussed and shared in the Slack channel.

Fixes #7278

### CODE REVIEW

File: communication/contributor-comms/storytelling-resources/sig-spotlight-blog-process.md [hunk 1]
```diff
@@ -1,10 +1,27 @@
...
+    * Create a Google Docs file with some initial questions, and ask them to write answers, and suggest additional topics if needed; we generally recommend doing the interview asynchronously via a Google Doc because written responses are easier to work with. The SIG-Multicluster one was done with a recorded zoom interview but it took a long time to complete.
+    * We have [example topics](example-topics.md) available for you to use, plus the existing articles that serve as a reference.
+    * You can either write some initial questions, get answers, and stop there, or based on the answers add new ones.
+    * Consider that you're not bound by your initial questions: it's very common for an answer to cover things that would be even better as a reply to a question that isn't there. You can add that questions and refactor answers to improve the flow, as long as the final version is reviewed and approved by the SIG person being interviewed.
+5. With that final version, create a markdown file in hackmd.io, based on the blog markdown template, and share it with the Contribex Comms Blogging lead and the community (the `#contribex-comms` channel).
```
Comment (chris-short) [inline communication/contributor-comms/storytelling-resources/sig-spotlight-blog-process.md:16]: #sig-contribex-comms is the channel name.

**Review by chris-short — CHANGES_REQUESTED** [review]
One nit.

File: communication/contributor-comms/storytelling-resources/sig-spotlight-blog-process.md (hunk 1 above)
Comment (fsmunoz) [reply communication/contributor-comms/storytelling-resources/sig-spotlight-blog-process.md]: Good catch, fixed!

**Pull request merged**